web/src/features/fireWeather/components/tables/SortableTableByDatetime.tsx

```diff
@@ -78,6 +136,68 @@ function SortableTableByDatetime<R extends WeatherValue>(props: Props<R>) {
     setOrder(order === 'asc' ? 'desc' : 'asc')
   }
 
+  const minMaxValuesToHighlight: MinMaxValues = {
+    relative_humidity:
+      _.minBy(props.rows, 'relative_humidity')?.relative_humidity ?? null,
+    wind_speed: _.maxBy(props.rows, 'wind_speed')?.wind_speed ?? null,
+    temperature: {
+      min: _.minBy(props.rows, 'temperature')?.temperature ?? null,
+      max: _.maxBy(props.rows, 'temperature')?.temperature ?? null
+    },
+    // depending on the table, the precip label may be 'precipitation',
+    // 'delta_precipitation', or 'total_precipitation' - need to account for all
+    precipitation:
+      _.maxBy(props.rows, 'precipitation')?.precipitation ??
+      _.maxBy(props.rows, 'delta_precipitation')?.delta_precipitation ??
+      _.maxBy(props.rows, 'total_precipitation')?.total_precipitation ??
+      null
+  }
+
+  const rowIds: RowIdsOfMinMaxValues = {
+    relative_humidity: [],
+    precipitation: [],
+    wind: [],
+    max_temp: [],
+    min_temp: []
+  }
+
+  rowsSortedByDatetime.forEach((row, idx) => {
+    if (
+      row.relative_humidity?.toFixed(RH_VALUES_DECIMAL) ===
+      minMaxValuesToHighlight.relative_humidity?.toFixed(RH_VALUES_DECIMAL)
+    ) {
+      rowIds['relative_humidity'].push(idx)
+    }
+    if (
+      row.precipitation?.toFixed(PRECIP_VALUES_DECIMAL) ===
```

Comment:
I think we need to exclude 0. If the max precip is 0, then we just have all the rows highlighted. I know it's mathematically correct, but it makes for a weird display i.m.h.o.